packages/login/src/vaadin-lit-login-form.js

```diff
@@ -0,0 +1,98 @@
+/**
```

Comment:
Noticed a small difference in how the login overlay dev page works when comparing the Lit and Polymer versions. With Lit version, the "Log in" button seems to be disabled when I re-open the login overlay whereas with the Polymer version it's always enabled:

https://github.com/vaadin/web-components/assets/1222264/8577d34d-102b-428d-afac-82f770dc7f67

Reply:
Thanks, I will look into it. Seems like we are missing a test for re-opening.

Reply:
@tomivirkki Fixed. It was a missing two-way binding for `disabled-changed` event 🙈